tests/Kernel/CallForTendersRenderTest.php

```diff
@@ -29,7 +29,6 @@ class CallForTendersRenderTest extends ContentRenderTestBase {
     'composite_reference',
     'oe_time_caching',
     'oe_content_reference_code_field',
-    'oe_content_departments_field',
```

Comment:
Can it be fixed already by the constraint of oe_content?

Reply:
No, this module and the below install configs removed are done in the ContentRenderTestBase since multiple content types tests need them.